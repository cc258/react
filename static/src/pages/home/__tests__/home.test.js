import React from 'react'
import {
	render,
	fireEvent,
	waitForElement,
	cleanup,
} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import { IntlProvider } from 'react-intl'
import Home from '../home'

afterEach(cleanup)

describe('<Home />', () => {
	test('should display a blank login form, with remember me checked by default', async () => {
		const { findByTestId, container } = render(
			<IntlProvider locale="en">
				<Home />
			</IntlProvider>
		)
		const home = await findByTestId('home')

		expect(container).toMatchSnapshot()
		expect(home).toBeInTheDocument()
		// expect(findByTestId('home')).toHaveTextContent('hello');
	})
})
