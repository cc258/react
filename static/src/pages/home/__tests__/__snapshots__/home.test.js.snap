// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To do Test: should render home 1`] = `
<div
  className="pages home"
  id="home"
>
  <h1>
    In the end, 
  </h1>
  <h2>
    everything will have a happy ending. 
  </h2>
  <h3>
    If it's not happy, 
  </h3>
  <h4>
    then it's not the end.
  </h4>
</div>
`;
